examples/computer_vision/unets_tf_keras/model_def.py

```diff
@@ -104,7 +105,11 @@ def build_model(self):
         model = self.context.wrap_model(model)
 
         # Create and wrap optimizer.
-        optimizer = tf.keras.optimizers.Adam()
+        # TODO MLG-443 Migrate from legacy Keras optimizers
+        if version.parse(tf.__version__) >= version.parse("2.11.0"):
+            optimizer = tf.keras.optimizers.legacy.Adam()
+        else:
+            optimizer = tf.keras.optimizers.Adam()
```

Comment:
Whereas I agree that for fixtures in e2e tests having this conditionality is the way to go, I am not sure it is appropriate here.